README.md

```diff
@@ -54,7 +54,7 @@ The list will be updated constantly and currently contains:
 To use the factory, some date must be provided in order to set it up:
 
 	MambuAPIServiceFactory serviceFactory = MambuAPIServiceFactory.getFactory(
-		"mydomain.mambu.com", "username", "password");
+		Protocol.HTTPS, "mydomain.mambu.com", "username", "password");
```

Comment:
I suggest we do not change the README file as Mambu continues to support only HTTPS connections and we should encourage users to use HTTPS only. Otherwise this may create a wrong impression. And we should continue providing all the same default methods which are using HTTPS.